test/src/main/java/org/apache/accumulo/test/MetaSplitIT.java

```diff
@@ -136,6 +166,15 @@ public void testMetadataTableSplit() throws Exception {
     }
   }
 
+  // Count the number of entries that can be read in the Metadata table
+  // This verifies all the entries can still be read after splits/merges
+  // when ranged files are used
+  private void verifyMetadataTableScan(AccumuloClient client) throws Exception {
+    // There should be 50 entries, 5 entries per Tablet
+    assertEquals(50,
+        client.createScanner(MetadataTable.NAME, Authorizations.EMPTY).stream().count());
```

Comment:
Instead of doing a count, this could verify some of the data in the metadata table.  This may also make the test less sensitive changes in the metadata table (like the lock column may be remove and that would break the count).

```suggestion
    var tables = client.tableOperations().tableIdMap();
    var expectedExtents = tables.entrySet().stream().filter(e->!e.getKey().startsWith("accumulo.")).map(Map.Entry::getValue).map(TableId::of).map(tid->new KeyExtent(tid, null,null)).collect(Collectors.toSet());
    assertEquals(10, expectedExtents.size());
    var ample = ((ClientContext)client).getAmple();

    try(var tablets = ample.readTablets().forLevel(Ample.DataLevel.USER).build()){
      for (var tablet : tablets) {
        assertTrue(expectedExtents.remove(tablet.getExtent()));
        assertNotNull(tablet.getDirName());
        assertNotNull(tablet.getLocation());
      }
    }

    // ensure all expected extents were seen
    assertEquals(0, expectedExtents.size());

```

Reply:
Thanks I'll make that change